plugins/oidc-auth/frontend/views/shared/_header_global.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="container global-header">
+  <nav class="navbar">
+    <div class="container">
+
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".nav-global-collapse">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+      </div>
+
+      <% if !session[:user] %>
+        <button type="button" class="btn btn-default pull-right btn-collapse-login navbar-btn" data-toggle="collapse" data-target=".nav-global-collapse">
+          <%= I18n.t("navbar.login") %>
+        </button>
+      <% end %>
+      <div class="navbar-collapse nav-global-collapse collapse navbar-right navbar-default">
+        <ul class="nav pull-right navbar-nav navbar-login">
+          <%= render "shared/header_user" %>
```

Comment:
This may have the logout to override.

Reply:
See https://github.com/mlibrary/aspace-containerization/pull/4/commits/4ab24da1f78fe5798fafc560184854bd29b63adf